earthpy/tests/test_bytescale.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import numpy as np
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def byte_arr():
+    return np.random.randint(300, size=(10, 10))
+
+
+def test_high_val_range():
+    """A 16 bit int arr with values at the end of the range, should be scaled properly.
+
+    Note that this test explicitly hits the user case of someone providing an array of a
+    dtype at the end of it's range of values. Bytescale should not fail in this case.
+    """
+
```

Comment:
Good place for a comment to explain where these magic numbers come from
```suggestion
    # The range of int16 values in python is -32768 to 32767
```